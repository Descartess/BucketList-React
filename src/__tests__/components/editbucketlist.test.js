import React from 'react';
import renderer from 'react-test-renderer';
import EditBucketListForm from '../../components/editbucketlist';

describe('Edit bucketlist snap shot ', () => {
  const props = {
    onChangeBucketListName: jest.fn(),
    onChangeBucketListAge: jest.fn(),
    onShowEditBucketList: jest.fn(),
    putBucketList: jest.fn(),
    bucketlist_name: '',
    bucketlist_age: 2,
    showEditBucketList: false,
  };
  it('renders correctly', () => {
    const rendered = renderer.create(
      <EditBucketListForm {...props} />,
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
