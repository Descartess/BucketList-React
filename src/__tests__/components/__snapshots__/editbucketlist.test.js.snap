// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit bucketlist snap shot  renders correctly 1`] = `null`;
